security/docker/Dockerfile.node-agent-k8s

```diff
@@ -1,9 +1,12 @@
 # BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
 ARG BASE_DISTRIBUTION=default
 
+# ISTIO_VERSION is used to specify the version of the release
+ARG ISTIO_VERSION=latest
```

Comment:
I am not in love with the expansion of the top level makefile `VERSION` variable to `ISITO_VERSION` in the dockerfiles.  Unfortunately it seems necesary - atleast for the proxy image, to uniquely identify the istio version and the proxy version.